### ISSUE TITLE
Format of the datasets

### ISSUE THREAD

**Issue by albarron** [albarron/LumpSTS#11]
The training datasets, some coming from different editions of the task have multiple formats. For instance, the Arabic files include id, score and the texts, whereas the Spanish and English ones come in 2 files: one with scores and one with pairs of texts. The English-Spanish file includes extra columns with MT information.

We had adopted the format of the Arabic files because it had more information. Nevertheless, the 2017 test set sticks to the English files format. As a result, we have to switch all the files (e.g., generated folds) to the same format as the one used in the 2017 test files.

**Comment by albarron** [comment]
This issue has been fixed.
